charts/linkerd-crds/templates/policy/server.yaml

```diff
@@ -72,7 +72,9 @@ spec:
                   default: unknown
     - name: v1beta1
       served: true
-      storage: true
+      storage: false
+      deprecated: true
```

Comment:
what are the consequences of deprecating this?  will this produce a flood of warning messages in the policy controller when the CRD is upgraded (before the control plane is upgraded)?

Reply:
I have tried that. Did an upgrade of the CRDS only and did not notice any deprecation warnings.